charts/ssi-dim-wallet-stub/Chart.yaml

```
@@ -23,7 +23,7 @@ description: |
   A Helm chart to deploy SSI DIM wallet stub in kubernetes cluster
 type: application
 version: 0.1.1
```

I'd assume, also a new chart version needs be released
```suggestion
version: 0.1.2
```